### ISSUE TITLE
Uninstall during update uses wrong version

### ISSUE THREAD

**Issue by se35710** [lukesampson/scoop#3588]
```
PS> scoop update *
[..]
Uninstalling '<something>' (0.0.1)
Running uninstaller script...
Remove-Item : Cannot find path 'something_v0.0.2.exe' because it does not exist.
```

**Comment by wenmin92** [comment]
Please tell us which app throws this error messge.

**Comment by se35710** [comment]
In a manifest, use:
```
"uninstaller": {
   "script": "echo $version.exe"
}
```
The result during upgrade is `$version` in the uninstall step returnes `0.0.2` (per example above). Is there another variable with the old version to use in uninstall?

**Comment by wenmin92** [comment]
Oh, I see. But I think you needn't delete the file yourself, while uninstaller will delete the whole folder.

**Comment by niheaven** [comment]
`$version` is defined globally, so it is new version while uninstalling. Define `$version` in the function will solve the problem.

**Comment by rasa** [comment]
Fixed in 96fef5827d58998c1c86533fb8156932579354e2
